Black Coffee Fest

anagudelogu opened the pull request:
# In this Capstone Project I:

- Made a page for a fantasy electronic music festival following the design guidelines and the requirements of the project.

- [Here's a link to the video presentation.](https://www.loom.com/share/bcf95edfc713455d99f24b977388a607)

All kind of feedback is well received, please include optional suggestions, these help me improve my technical skills! :nerd_face:

anagudelogu committed:
Add mobile menu basic design

anagudelogu committed:
fix ESlint and Stylelint issues

anagudelogu committed:
Fix design conflicts between mobile and desktop versions

anagudelogu committed:
Merge pull request #9 from anagudelogu/js-mobile-menu

Js mobile menu

anagudelogu committed:
Add dynamically each artist card to the home page.

anagudelogu committed:
Change the structure of the artist object

Changed the type of 'photo' property  of the artist object from array to string.

anagudelogu committed:
Merge pull request #10 from anagudelogu/js-dynamic-artists

Js Dynamic artist section

anagudelogu committed:
Add page's content

anagudelogu committed:
Fix linter issues

anagudelogu committed:
Merge pull request #11 from anagudelogu/content

Add page's content

anagudelogu committed:
Fix styling issues, feature section

anagudelogu committed:
Fix styling issues , feature section, desktop devices

anagudelogu committed:
Merge pull request #12 from anagudelogu/func

Minor style issues

anagudelogu committed:
Add Readme file to the project

anagudelogu committed:
Merge pull request #13 from anagudelogu/documentation

Add Readme file to the project

RolandM99 commented on index.html:
```diff
@@ -93,7 +87,7 @@ <h2 class="partner__title section-title"> Partner</h2>
     <footer>
-        <p>Capstone project 1 - made by: <br>Andres Agudelo Guzman.</p>
+        <p>Capstone project 1 - made by: <br>Andres Agudelo Guzman.<br> *This is a fantasy music festival web page, made for educational purposes*</p>
     </footer>
```
- [ ] **REQUIRED**: Our website footer should contain the logo aligned to the left and the text aligned to the right. Please, Kindly fix it.
### Behance template: 
![image](https://user-images.githubusercontent.com/71874425/163288358-95d5e72d-61fd-449c-8a7e-c93a70cb9fae.png)
### Your design:
![image](https://user-images.githubusercontent.com/71874425/163288422-723b63b8-c3fc-4d98-927a-da7d8c0f7de2.png)
- Make sure that your website UI fit the design guidelines provided on behance

RolandM99 commented on styles.css:
```diff
@@ -93,7 +145,6 @@ a:active {
 .hero__container {
   background-image: url(./images/mobile_intro_bg.png);
   background-size: contain;
```
Your template  | Original Design 
------------- | -------------
![image](https://user-images.githubusercontent.com/71874425/163292140-6b070d3c-5a5e-4ebb-8b52-2e82da2a6793.png) | ![image](https://user-images.githubusercontent.com/71874425/163292227-683efe18-6f4a-41de-874c-2ce64f3221ac.png)

- [ ] Make sure that our website pages look almost identical to the original design. The fonts (color, text,...) should be the same as the original design.
- [ ] Instead of using an `img` tag for the page title, kindly use an `h1` tag. the `h1` tag serves as a title to describe what your webpage is about. You can change this on line 39 in your HTML code.

RolandM99 commented on script.js:
```diff
@@ -0,0 +1,97 @@
+
+function toggleMenu() {
+  hamburger.classList.toggle('active');
+  menu.classList.toggle('active');
```
- [ ]  Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is an example of the syntax of an ES6 arrow function...
```javascript
  const toggleMenu = () => {
     // code here
  }
```

RolandM99 commented on about.html:
```diff
@@ -72,7 +74,8 @@ <h2 class="partner__title section-title"> Partner</h2>
         </section>
     </main>
     <footer>
-        <p>Capstone project 1 - made by: <br>Andres Agudelo Guzman.</p>
+        <p>Capstone project 1 - made by: <br>Andres Agudelo Guzman.<br> *This is a fantasy music festival web page, made for educational purposes*</p>
     </footer>
```
- [ ] Make sure that the fonts (background color) for your footer section on the about page fit the original [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

RolandM99 commented on styles.css:
```diff
@@ -74,15 +77,64 @@ a:active {
+.navbar__hamburger.active {
+  display: none;
+}
+
+.navbar__links {
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
```
- [ ] The nav link menu should have the similar spacing as the original design. 
![image](https://user-images.githubusercontent.com/71874425/163296135-5b93c59a-b169-4d61-99ce-847afa9c3d58.png)
- Please, Kindly fix it.

RolandM99 reviewed (changes requested):
Hi  @anagudelogu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)

### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please, Instead of adding your video presentation link in your PR description, add it in the README text.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

anagudelogu committed:
Fix requirements from review:

1. Changed the image title for an h1 title.
2. Changed ES5 functions to ES6 arrow ones.
3. Changed the background color and font color of footer section.
4. Fixed spacing of the navbar.

anagudelogu committed:
Fix linter issues

anagudelogu committed:
Update README file with video presentation link

Danieldotcomcoder reviewed (approved):
Hello @anagudelogu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

![comp-2](https://user-images.githubusercontent.com/87448628/159119684-e86473c4-9b60-43a3-90f2-54048e841394.gif)
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
